model/articles.model.js

```diff
@@ -0,0 +1,38 @@
+const db = require("../db/connection");
+
+exports.fetchAllArticles = () => {
+  return db
+    .query(
+      `SELECT articles.*,CAST(COUNT(comments.article_id) AS INTEGER) AS comment_count FROM comments
+  FULL OUTER JOIN articles ON articles.article_id=comments.article_id
```

Comment:
should ideally be a LEFT JOIN for getting comment count (sorry I know that's a diff PR 😅)